# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/IPCNavigationPreloadState.ipdlh
# Commit: 679bed36bc0b
# Full Hash: 679bed36bc0b4dbca32def451b6385e5a898368b
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2021-06-10 09:49:36
# Regressor Bug: 1564235
# File Overlap Count: 2
# Description:
#   Bug 1564235 - P3 Create IPC for NavigationPreload in PServiceWorkerRegistration.ipdl. r=dom-worker-reviewers,asuth
#   
#   This patch implements the IPC for NavigationPreload APIs.
#   The IPC methods are created on PServiceWorkerRegistration.
#   
# ==============================================================================

diff -r 789ce1a19e9f -r 679bed36bc0b dom/serviceworkers/IPCNavigationPreloadState.ipdlh
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/serviceworkers/IPCNavigationPreloadState.ipdlh	Thu Jun 10 00:48:11 2021 +0000
@@ -0,0 +1,16 @@
+/* -*- Mode: C++; c-basic-offset: 2; indent-tabs-mode: nil; tab-width: 8 -*- */
+/* vim: set sw=4 ts=8 et tw=80 ft=cpp : */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+namespace mozilla {
+namespace dom {
+
+struct IPCNavigationPreloadState {
+  bool enabled;
+  nsCString headerValue;
+};
+
+} // namespace dom
+} // namespace mozilla